### ISSUE TITLE
Discharge alert with incomplete discharge while picking up other charges

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#311]
In #310 @KatieWoe said

> Finally, you can keep adding charges while discharge occurs, and this seems to confuse the screen reader when it tries to say how many charges were lost.

This case may not be handled well. Ill take a look

**Comment by jessegreenberg** [comment]
@KatieWoe could you please elaborate on the problem you observe? Here are the steps I am taking to try to reproduce:
- Rub foot on rug until enough charges are collected to discharge.
- While electrons are moving to doorknob, rub foot on rug to add more charges to the body. Do this late enough in the electron discharge so that the new charges don't immediately move to the doorknob.
- With the new charge in the body, the AT says something like
> Discharge occurred. Electrons on body decreased from 22 to 1 with hand at position -1, just above doorknob.

I think this is a correct description based on the state of the simulation. @KatieWoe is this what you observed or did something else happen?

**Comment by KatieWoe** [comment]
Not exactly what I meant, sorry for being unclear.
1. Rub foot on rug until enough charges are collected to discharge.
2. As discharge starts, continue to rub foot on rug, continuously creating charges that flow out in the same discharge.
3. After this has occurred for some time listen to the description.  If I remember correctly it said something similar to what it should at the start, but cut off when trying to read the numbers.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, was this with JAWS specifically or all AT?

**Comment by KatieWoe** [comment]
@jessegreenberg I only noticed it JAWS.  I can't remember if I tried it on any other screenreaders, so it might be on all of them.  I can check NVDA really quick and get back to you on that.

**Comment by jessegreenberg** [comment]
@KatieWoe and I discussed over Slack. Keyboard interaction with NVDA isn't performant enough to support this case on the Windows platforms available to us. I will try to reproduce the case mentioned in https://github.com/phetsims/john-travoltage/issues/311#issuecomment-426296616 on JAWS on my laptop.

**Comment by KatieWoe** [comment]
Closing for @jessegreenberg

**Comment by jessegreenberg** [comment]
@KatieWoe and I re-tested. I am generally not getting any alerts after moving the leg after a discharge, but this makes sense due to the fact that AT will silence all previous alerts after an interaction. I believe this can be closed.
